src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {Route} from 'react-router-dom';
 
 const App = () => (
   <div>
-    <Navigation />
+    <Navigation>
+    <Route exact path="/" component={Home} />
+    <Route exact path="/about" component={About} />
+    <Route exact path="/contact" component={Contact} />
+    </Navigation>
```

Comment:
Here, you just need to call your navigation component, it looks like you're more trying to use it as the html <nav> tag. If it works, that's totally fine, but your code would be DRY-er if you just called <Navigation /> as a self-closing component call.